umake/frameworks/ide.py

```diff
@@ -473,3 +474,83 @@ def post_install(self):
                                                      categories=categories))
         if not self.was_in_arduino_group:
             UI.delayed_display(DisplayMessage(_("You need to logout and login again for your installation to work")))
+            
+class BaseNetBeans(umake.frameworks.baseinstaller.BaseInstaller):
+    """The base for all Netbeans installers."""
+    
+    BASE_URL = "http://download.netbeans.org/netbeans/8.0.2/final/"
+
+    def __init__(self, category, version="8.0.2", flavour=""):
```

Comment:
So, same here, I would really like to remove any kind of "version", especially in description, and so on. We always pull latest, whatever it is at the current moment (I know that eclipse Luna is an exception, but I want to remove this, see some other pull requests :))